odxtools/minmaxlengthtype.py

```diff
@@ -34,11 +34,11 @@ def __post_init__(self):
     def dct_type(self) -> DctType:
         return "MIN-MAX-LENGTH-TYPE"
 
-    def __termination_character(self):
+    def __termination_character(self) -> bytes:
         """Returns the termination character or None if it isn't defined."""
```

Comment:
docs don't match implementation

Reply:
docs says "Returns the termination character or None", I don't see a return None in the code.

Reply:
ah, you mean the `None`? right, fixed: c91ebaa7877d94fff2f10ae10900bc194813e418 (I also renamed this to `__termination_sequence`